osbs/api.py

```diff
@@ -425,33 +423,68 @@ def _create_build_directly(self, build_request, unique=None):
 
         return BuildResponse(self.os.create_build(build_json).json(), self)
 
+    @retry_on_conflict
+    def _get_or_create_imagestream(self, imagestream_name, build_request):
+        insecure = False
+        source_registry_uri = RegistryURI(build_request.source_registry['url']).docker_uri
+        source_registry_insecure = build_request.source_registry.get('insecure', False)
+
+        docker_image_repo = ImageName.parse(build_request.base_image)
+        if build_request.organization:
```

Comment:
This must be done conditionally based on the _registry_ configuration. You mentioned this to me in IRC, just adding a comment here so I don't forget it :)

Reply:
already fixed :)